.github/assets/release/aur/PKGBUILD.template.sh

```diff
@@ -0,0 +1,24 @@
+# Maintainer: Jérôme Deuchnord <jerome@deuchnord.fr>
+
+pkgname=kosmorro
+pkgver=___VERSION___
+pkgrel=1
+pkgdesc='A program that computes the ephemerides'
+depends=('python>=3.5.0' 'python<4.0.0' 'python-skyfield>=1.14' 'python-skyfield<2.0.0' 'python-numpy>=1.17.0' 'python-numpy<2.0.0' 'python-tabulate')
```

Comment:
- Update the dependencies according to current PKGBUILD
- Add a new optional dependencies: `python-latex` and `texlive-core` (see #44)